### PR TITLE
Prepare v1.8.0 release

### PR DESCRIPTION
**Pull request by ekamin-hpe**
* Declare version 1.8.0 in the changelog with previously-unreleased changes.
* Update comparison links at the bottom of the change log.

## Pull Request Checklist

- [x] Target branch correct
- [X] Testing is appropriate and complete, if applicable
- [X] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable
- [X] Is a new version being released? Update https://github.com/Cray-HPE/cray-product-catalog/wiki/CSM-Compatibility-Matrix

### CODE REVIEW

**Review by haasken-hpe — DISMISSED** [review]

**Review by jack-stanek-hpe — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased]
 
+## [1.8.0] - 2022-12-20
+
+### Added
+- Added an environment variable `REMOVE_ACTIVE_FIELD`. When set, the `catalog_update`
+  script will remove the 'active' field for all versions of the given product.
+
+### Changed
+
```
Comment (jsollom-hpe) [inline CHANGELOG.md:17]: Delete extra line here.

**Review by jsollom-hpe — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ekamin-hpe) [reply CHANGELOG.md:17]: Hmm, didn't notice that before. Are you sure that I shouldn't actually have added blank lines under `### Added` and `### Fixed`? It looks like that is the pattern for older entries here in the change log.

Comment (ekamin-hpe) [reply CHANGELOG.md:17]: And for that matter, between the individual entries as well?

Comment (ekamin-hpe) [reply CHANGELOG.md:18]: Adjusted spacing in 1aa61e8b29f031626d62b5478cd8bffc9bcf14bf

**Review by jsollom-hpe — APPROVED** [review]

**Review by haasken-hpe — APPROVED** [review]

**Pull request merged**